### ISSUE TITLE
Agent Forwarding

### ISSUE THREAD

**Issue by alasdairdc** [connectbot/connectbot#378]
Is it possible to allow Agent Forwarding when connected to a server like the command `ssh -At my_host`?  I tried the `Use SSH Auth agent` but this doesn't work / seems to be for something else.

**Comment by RoundSparrow** [comment]
Perhaps not related - but what exactly does the "Use SSH Auth agent" option do?  What use is it, what problem does it solve?

**Comment by kruton** [comment]
That's what it does. What type of key did you have loaded in ConnectBot? I just noticed that EC keys were not being handled correctly.

**Comment by alasdairdc** [comment]
Yes I was trying this with EC keys.  Will update from master / rebuild and test

**Comment by alasdairdc** [comment]
Logging into the server and running `ssh-add -L` results in 

    The agent has no identities.

Help identifies itself as `v1.9.0-alpha1-26-g8a904e8` with the host configured as `Use SSH auth agent` and `require confirmation`.

**Comment by kruton** [comment]
It will only show loaded keys there, so if your key is show with a "locked" icon it will not list it as an identity.

**Comment by alasdairdc** [comment]
Ah okay cool.  Just tested, and it shows if the key is explicitly unlocked before connecting,

As an aside on other hosts 'AgentForwarding is disabled and I normally (on Laptop) connect using the proxy command:

    ProxyCommand ssh -q -W %h:%p jumper

Due to the firewall rules no abnormal ports can be opened.  I'm sure this is possible in ConnectBot using port forwards but am unsure how.  Any pointers would be great

**Comment by saschaarthur** [comment]
can confirm that it does not work..

**Comment by workingmachine** [comment]
It does work with connectbot f-droid version 1.9.9-oss.
I have two EC key, both generated with PuTTYgen and converted to OpenSSH keys (new format). One key is for connection authentication and the other key is for sudo authentication.
For sudo authentication the settings „Use SSH auth agent“ as well as „require conformation“ are enabled.
Every time when I use sudo command a conformation dialog with yes and no button appears.

Thank you for that great terminal / Secure Shell.
